test/MUXControls.Test/Infra/TestHelpers.cs

```diff
@@ -163,6 +163,21 @@ public TestSetupHelper(ICollection<string> testNames, string languageOverride =
 
                         Log.Comment("__TestContentLoadedCheckBox checkbox checked, page has loaded");
 
+
+                        ToggleButton tb = new ToggleButton(FindElement.ById("__InnerFrameInLabDimensions"));
+                        if(tb.ToggleState != ToggleState.On && shouldRestrictInnerFrameSize)
+                        {
+                            Log.Comment("toggling the __InnerFrameInLabDimensions toggle button to On.");
```

Comment:
Should the default in the app be "On"? That way we save a little time when running the tests. And the app looks "correct" by default when launched manually. It might make the test app look a little ugly by default - but it is already ugly! :)

Reply:
agreed and fixed